docs/apache-airflow-providers-amazon/operators/eks.rst

```diff
@@ -0,0 +1,265 @@
+and ``EKSDescribeNodegroupOperator`` to verify creation, then ``EKSDeleteNodegroupOperator``
+to delete the nodegroup.
+
+Prerequisites
+"""""""""""""
+
+An AWS IAM role with the following permissions:
+
+  "ec2.amazon.aws.com" must be in the Trusted Relationships
+  "AmazonEC2ContainerRegistryReadOnly" IAM Policy must be attached
+  "AmazonEKSWorkerNodePolicy" IAM Policy must be attached
+
+Defining tasks
+""""""""""""""
+
+In the following code we create a new Amazon EKS Managed Nodegroup.
+
+.. exampleinclude:: /../../airflow/providers/amazon/aws/example_dags/example_eks_create_nodegroup.py
+    :language: python
+    :start-after: [START howto_operator_eks_create_nodegroup]
+    :end-before: [END howto_operator_eks_create_nodegroup]
+
+
+.. _howto/operator:EKSListNodegroupsOperator:
+.. _howto/operator:EKSDescribeNodegroupOperator:
+
+Listing and Describing Amazon EKS Clusters
+-------------------------------------------
+
+Defining tasks
+""""""""""""""
+
+In the following code we retrieve details for a given Amazon EKS nodegroup.
+
+.. exampleinclude:: /../../airflow/providers/amazon/aws/example_dags/example_eks_create_nodegroup.py
+    :language: python
+    :start-after: [START howto_operator_eks_describe_nodegroup]
+    :end-before: [END howto_operator_eks_describe_nodegroup]
+
+
+In the following code we list all Amazon EKS Nodegroups in a given EKS Cluster.
+
+.. exampleinclude:: /../../airflow/providers/amazon/aws/example_dags/example_eks_create_nodegroup.py
+    :language: python
+    :start-after: [START howto_operator_eks_list_nodegroup]
+    :end-before: [END howto_operator_eks_list_nodegroup]
+
+
+.. _howto/operator:EKSDeleteNodegroupOperator:
+
+Deleting Amazon EKS Managed Nodegroups
+--------------------------------------
+
+Defining tasks
+""""""""""""""
+
+In the following code we delete an Amazon EKS nodegroup.
+
+.. exampleinclude:: /../../airflow/providers/amazon/aws/example_dags/example_eks_create_nodegroup.py
+    :language: python
+    :start-after: [START howto_operator_eks_delete_nodegroup]
+    :end-before: [END howto_operator_eks_delete_nodegroup]
+
+
+Creating Amazon EKS Clusters and Node Groups Together
+------------------------------------------------------
+
+Purpose
+"""""""
+
+This example dag ``example_eks_create_stack.py`` demonstrates using
+``EKSCreateClusterOperator`` to create an Amazon EKS cluster and underlying
+Amazon EKS node group in one command.  ``EKSDescribeClustersOperator`` and
+``EKSDescribeNodegroupsOperator`` verify creation, then ``EKSDeleteClusterOperator``
+deletes all created resources.
+
+Prerequisites
+"""""""""""""
+
+  "ec2.amazon.aws.com" must be in the Trusted Relationships
+  "eks.amazonaws.com" must be added to the Trusted Relationships
+  "AmazonEC2ContainerRegistryReadOnly" IAM Policy must be attached
+  "AmazonEKSClusterPolicy" IAM Policy must be attached
+  "AmazonEKSWorkerNodePolicy" IAM Policy must be attached
```

Comment:
```suggestion
  `ec2.amazon.aws.com` must be in the Trusted Relationships
  `eks.amazonaws.com` must be added to the Trusted Relationships
  `AmazonEC2ContainerRegistryReadOnly` IAM Policy must be attached
  `AmazonEKSClusterPolicy` IAM Policy must be attached
  `AmazonEKSWorkerNodePolicy` IAM Policy must be attached
```
To avoid spelling errors.

Reply:
Does the spell check know to ignore anything inside double back ticks?

Reply:
Fixed in 616d249